bundles/org.openhab.binding.network/pom.xml

```diff
@@ -34,7 +34,7 @@
     <dependency>
       <groupId>commons-net</groupId>
```

Comment:
@wborn isn't this superfluous too? Ather add-ons depending on `apache.commons.net` do not include it their POMs. I guess it was only necessary because it is a dependency of `fr.bmartel`.

Reply:
Actually I was asking myself the same, i.e. why this explicit dependency even exists :-)

Reply:
The `NetworkUtils` class uses `SubnetUtils` from commons-net. Whenever add-ons have `org.apache.commons.net` imports in their code, you want to make sure its using the same version as provided by the core Karaf feature. `fr.bmartel` depends on 3.6 which would then be the version because it is a transitive dependency. Normally you'd exclude commons-net from `fr.bmartel` to get the version we want to use, but because the core compile BOM does not define a compile dependency you need to define it here.

Maybe we can also define a commons.net version property in a core BOM (but not add the dependency). Then we can still keep the version in sync if we want to add it in add-ons that currently use it... but it may prevent commons-net from being used in new add-on contributions.